packages/discovery-react-components/src/components/SearchInput/SearchInput.tsx

```diff
@@ -122,7 +127,8 @@ const SearchInput: FC<SearchInputProps> = ({
     const prefix = valueArray.pop();
     const completionValue = !!completions ? completions[i] : prefix;
     valueArray.push(completionValue || '');
-    setValue(`${valueArray.join(splitSearchQuerySelector)}${splitSearchQuerySelector}`);
+    const actualValue = `${valueArray.join(splitSearchQuerySelector)}${splitSearchQuerySelector}`;
```

Comment:
What is the purpose of defining this variable instead of just passing all of this into the `setValue` function?

Reply:
Hmm, this seems to be left over changes that I forgot to undo when redoing some work with this component.  I was using the `setValue` twice before, so it made sense then to make a `const`.  Will change back, since it's not necessary anymore.